front/app/containers/Admin/projects/project/analysis/TopBar/index.tsx

```diff
@@ -32,7 +32,7 @@ const TopBar = () => {
   const { formatMessage } = useIntl();
 
   const goBack = () => {
-    if (analysis?.data.attributes.participation_method === 'survey') {
+    if (analysis?.data.attributes.participation_method === 'native_survey') {
       clHistory.push(`/admin/projects/${projectId}/native-survey/results`);
```

Comment:
Not part of this PR, I know, but one better would be to go to the actual results page, of the survey, which I assume needs the phase in the url?

Reply:
Didn't consider the phase scenario here.. Thanks!